govcd/sample_govcd_test_config.yaml

```diff
@@ -141,6 +141,11 @@ ova:
   #
   # The chunked ova (vmdk inside are split) for tests
   ovaChunkedPath: ../test-resources/template_with_custom_chunk_size.ova
+  #
+  # The ova with multi VMs for tests.
+  ovaMultiVmPath: ../test-resources/3VM.ova
```

Comment:
Suggesting to clarify and remove capital letters: `3VM.ova` -> `vapp_with_3_vms.ova`

Reply:
done

Reply:
You have double ending there: `vapp_with_3_vms.ova.ova`

Reply:
it's not there, it was file renaming issue